less/mixins.less

```diff
@@ -1 +1,10 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+
+.btn (@color, @background-color){
+    color: @porcelain;
+    background-color: @btn-bg-color;
```

Comment:
you will want to use an argument passed into one of the parameters of your mixin typically.  otherwise, no point in having parameters on the mixin.